schema/proto3/port.proto

```diff
@@ -28,13 +28,14 @@ message PortConfiguration {
     
     NetworkType network_type = 2;
     string project_id = 3;
-    string network_id = 4;
-    string id = 5;
-    string name = 6;
-    string network_ns = 7;
-    string mac_address = 8;
-    string veth_name = 9;
-    bool admin_state_up = 10;
+    string subnet_id = 4;
+    uint32 mq_subscription_index = 5;
+    string id = 6;
+    string name = 7;
+    string network_ns = 8;
+    string mac_address = 9;
+    string veth_name = 10;
+    bool admin_state_up = 11;
```

Comment:
Do we need all of these fields for port given that we have better picture of OVS?

Reply:
network_ns - was used during our previous k8s integration, we may need it for later k8s integration
mac_address - for virtual mac
veth_name - don't think we need that for OVS, I have moved that to mizar specific session
admin_state_up - neutron specific field, we want to keep it

Reply:
Cool thanks!